<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_19">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>248</y>
      <width>74</width>
      <height>53</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #228B22;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}

QPushButton:pressed
{
	background-color: #778899;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}
</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>248</y>
      <width>74</width>
      <height>53</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #800080;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}

QPushButton:pressed
{
	background-color: #778899;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}
</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>248</y>
      <width>74</width>
      <height>53</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #800080;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}

QPushButton:pressed
{
	background-color: #778899;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>12</y>
      <width>394</width>
      <height>53</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	background-color: #FFFF00;
	border: 1px solid gray;
	color: #FF4500;
}</string>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>71</y>
      <width>74</width>
      <height>53</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #800080;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}

QPushButton:pressed
{
	background-color: #778899;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}
</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>74</width>
      <height>53</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #800080;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}

QPushButton:pressed
{
	background-color: #778899;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}
</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>71</y>
      <width>74</width>
      <height>53</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #228B22;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}

QPushButton:pressed
{
	background-color: #778899;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}
</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>74</width>
      <height>53</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #800080;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}

QPushButton:pressed
{
	background-color: #778899;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}
</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_16">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>71</y>
      <width>74</width>
      <height>53</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #228B22;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}

QPushButton:pressed
{
	background-color: #778899;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}
</string>
    </property>
    <property name="text">
     <string>M+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>189</y>
      <width>74</width>
      <height>53</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #800080;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}

QPushButton:pressed
{
	background-color: #778899;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}
</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>71</y>
      <width>74</width>
      <height>53</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #800080;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}

QPushButton:pressed
{
	background-color: #778899;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}
</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_18">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>189</y>
      <width>74</width>
      <height>53</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #228B22;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}

QPushButton:pressed
{
	background-color: #778899;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}
</string>
    </property>
    <property name="text">
     <string>M</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>248</y>
      <width>74</width>
      <height>53</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #800080;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}

QPushButton:pressed
{
	background-color: #778899;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}
</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_15">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>248</y>
      <width>74</width>
      <height>53</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #228B22;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}

QPushButton:pressed
{
	background-color: #778899;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}
</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>71</y>
      <width>74</width>
      <height>53</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #800080;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}

QPushButton:pressed
{
	background-color: #778899;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}
</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>74</width>
      <height>53</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #228B22;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}

QPushButton:pressed
{
	background-color: #778899;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}
</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_17">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>130</y>
      <width>74</width>
      <height>53</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #228B22;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}

QPushButton:pressed
{
	background-color: #778899;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}
</string>
    </property>
    <property name="text">
     <string>M-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>189</y>
      <width>74</width>
      <height>53</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #800080;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}

QPushButton:pressed
{
	background-color: #778899;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}
</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_14">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>189</y>
      <width>74</width>
      <height>53</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #228B22;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}

QPushButton:pressed
{
	background-color: #778899;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>189</y>
      <width>74</width>
      <height>53</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #800080;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}

QPushButton:pressed
{
	background-color: #778899;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}
</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>74</width>
      <height>53</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #800080;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}

QPushButton:pressed
{
	background-color: #778899;
	border:1px solid #EEE8AA;
	padding: 5px;
	color: #00FFFF;
}
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
